### ISSUE TITLE
Possible security issue disclosure

### ISSUE THREAD

**Issue by miguelhdezsob** [phamleduy04/texas-dps-scheduler#34]
First of all, thanks for creating this, it is very helpful.

I think I just found a possible security issue on this repo, however, I do not want to report this openly as an Issue as all issues are public.

Is there an email I could contact you in privately? Alternatively you can reach out to me at miguelhdezsob@gmail.com

**Comment by phamleduy04** [comment]
Hi, I think now that we should disclose the security issue to everyone.

TL;DR: If you using a repl.it free plan, you might get your personal information in your `config.yml` leaked publicly. This includes firstname, lastname, last4SSN, email and phone number.

I'm really sorry about this situation and this it what I do to handle this issue:

- [x] Update app to handle repl.it secrets, which can't be viewed public 
- [x] Update wiki and Youtube video to show you guys how to use repl.it secrets.
- [x] Contact repl.it to delete all public repls that contain personal information. (Still have some left but replit did indeed deleted about 200+ repls)

![image](https://user-images.githubusercontent.com/32657584/230106630-d69a0b10-1368-4465-9620-cc7423cbd496.png)
